src/lib/components/home/artwork_rails/artwork_rail.tsx

```diff
@@ -203,29 +209,32 @@ export class ArtworkRail extends React.Component<Props & RelayPropsWorkaround, S
     }
   }
 
+  railStyle() {
+    const sideMargin = isPad ? 40 : 20
+    const style: any = { marginLeft: sideMargin, marginRight: sideMargin }
+
+    if (!this.hasStartedFetching()) {
```

Comment:
This doesn’t convey the intention well imo. The min height needs to be set while Relay has not finished fetching, not only while Relay has not started fetching yet.